source/platforms/github/_tests/_github_git.test.ts

```diff
@@ -20,15 +20,19 @@ export const requestWithFixturedContent = async (path: string): Promise<() => Pr
   Promise.resolve(readFileSync(`${fixtures}/${path}`, {}).toString())
 )
 
+const pullRequestInfoFilename = "github_pr.json"
+const masterSHA = JSON.parse(readFileSync(`${fixtures}/${pullRequestInfoFilename}`, {}).toString()).base.sha
```

Comment:
This could possibly be a bug with Jest's snapshot testing. :-/ Their Windows support is still pretty unstable AFAIK.